internal/cluster/cluster.go

```diff
@@ -263,8 +263,10 @@ func (m *Manager) reportValidationStatusChanged(ctx context.Context, c *common.C
 	for vCategory, vRes := range newValidationRes {
 		for _, v := range vRes {
 			if currentStatus, ok := m.getValidationStatus(currentValidationRes, vCategory, v.ID); ok {
-				if v.Status == ValidationFailure && currentStatus == ValidationSuccess {
-					m.metricAPI.ClusterValidationChanged(models.ClusterValidationID(v.ID))
+				if v.Status == ValidationFailure && currentStatus != ValidationFailure {
```

Comment:
can we alias `newStatus := v.Status` and rename `currentStatus` to `previousStatus`? The current naming gives me a headache

Reply:
I know :smile: 
Will do